test_conformance/events/main.cpp

```diff
@@ -24,39 +24,41 @@
 #endif
 
 test_definition test_list[] = {
-    ADD_TEST(event_get_execute_status),
-    ADD_TEST(event_get_write_array_status),
-    ADD_TEST(event_get_read_array_status),
-    ADD_TEST(event_get_info),
-    ADD_TEST(event_wait_for_execute),
-    ADD_TEST(event_wait_for_array),
-    ADD_TEST(event_flush),
-    ADD_TEST(event_finish_execute),
-    ADD_TEST(event_finish_array),
-    ADD_TEST(event_release_before_done),
-    ADD_TEST(event_enqueue_marker),
-#ifdef CL_VERSION_1_2
-    ADD_TEST(event_enqueue_marker_with_event_list),
-    ADD_TEST(event_enqueue_barrier_with_event_list),
-#endif
+    //    ADD_TEST(event_get_execute_status),
```

Comment:
I think these tests were only commented out for testing, and should be re-enabled.

Reply:
@bashbaug yes it was only for internal testing. Re-enabled again